### PR TITLE
Beaxy wallet ui

### PR DESCRIPTION
**Pull request by konstantin1390**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/investarena/redux/reducers/platformReducer.js [hunk 1]
```diff
@@ -122,3 +126,5 @@ export const getPlatformName = state => state.platformName;
 export const getAccountsMap = state => state.accountsMap;
 export const getCurrencySettings = state => state.currencySettings;
 export const getUserStatistics = state => state.userStatistics;
+export const getBeaxyWallet = state => state.userWallet;
```
Comment (milklim) [inline src/investarena/redux/reducers/platformReducer.js]: add userWallet to initial state

File: src/client/user/UserWallet.js [hunk 2]
```diff
@@ -144,8 +166,9 @@ class Wallet extends Component {
       cryptosPriceHistory,
       screenSize,
       guestBalance,
+      // beaxyBalance,
```
Comment (milklim) [inline src/client/user/UserWallet.js]: remove

File: src/investarena/redux/actions/platformActions.js [hunk 3]
```diff
@@ -2,6 +2,9 @@ import Cookies from 'js-cookie';
 import apiExtra from '../../apiExtra/Account';
 import { authorizeTokenSuccess } from './userActions';
 import { singleton } from '../../platform/singletonPlatform';
+import { GET_PERFORMERS_STATS } from './topPerformersActions';
```
Comment (milklim) [inline src/investarena/redux/actions/platformActions.js]: unused import

File: src/investarena/platform/umarkets.js [hunk 4]
```diff
@@ -160,6 +161,7 @@ export default class Umarkets {
     this.getUserStatistics();
     this.getUserRates();
     this.getCrossStatistics();
+    this.getCryptoCurrenciesDescription();
```
Comment (milklim) [inline src/investarena/platform/umarkets.js]: add todo: move to more suitable place

File: src/investarena/redux/actions/platformActions.js (hunk 3 above)
Comment (konstantin1390) [reply src/investarena/redux/actions/platformActions.js]: ++

File: src/client/user/UserWallet.js (hunk 2 above)
Comment (konstantin1390) [reply src/client/user/UserWallet.js]: ++

File: src/investarena/redux/reducers/platformReducer.js (hunk 1 above)
Comment (konstantin1390) [reply src/investarena/redux/reducers/platformReducer.js]: ++

File: src/investarena/platform/umarkets.js (hunk 4 above)
Comment (konstantin1390) [reply src/investarena/platform/umarkets.js]: will move on next PR

**Pull request merged**